ToDoApp/components/item/Item.js

```diff
@@ -1 +1,48 @@
-//File
+import React, { Component } from 'react';
+import {TouchableOpacity,Text, View, StyleSheet} from 'react-native';
+import CheckBox from '../checkBox/CheckBox';
+
+class Item extends Component {
+
+  render() {
+    return (   
+    <TouchableOpacity style={{backgroundColor: "white"}} onPress={this.props._onPress}>
```

Comment:
same about the styles thing here

Reply:
I have just changed that, in the next push it will be added.